src/2024h2/Rust-for-SciComp.md

```diff
@@ -72,67 +60,71 @@ algebra and we start to see more and more libraries in other languages use Rust
 
 **Owner:** ZuseZ4 / Manuel S. Drehwald
 
-Manuel S. Drehwald working 5 days/wk, sponsored by LLNL and the University of Toronto (UofT).
-He has a background in HPC and worked on a rust compiler fork, as well as an LLVM based autodiff tool 
-for the last 3 years during his undergrad. He is now in a research based Master Program. 
-Supervision and Discussion on the LLVM side with Johannes Doerfert and Tom Scogland.
+Manuel S. Drehwald is working 5 days per week on this, sponsored by LLNL and the University of Toronto (UofT). He has a background in HPC and worked on a Rust compiler fork, as well as an LLVM-based autodiff tool for the last 3 years during his undergrad. He is now in a research-based master's degree program. Supervision and discussion on the LLVM side will happen with Johannes Doerfert and Tom Scogland.
 
-Resources:
-Domain and CI for the autodiff work provided by MIT. Might be moved to the LLVM org later this year.
-Hardware for Benchmarks provided by LLNL and UofT.
-CI for the offloading work provided by LLNL or LLVM(?, see below).
+Resources: Domain and CI for the autodiff work is being provided by MIT. This might be moved to the LLVM org later this year. Hardware for benchmarks is being provided by LLNL and UofT. CI for the offloading work will be provided by LLNL or LLVM (see below).
 
 ### Support needed from the project
 
-* Discussion on CI: It would be nice to test the Offloading support on at least all 3 mayor GPU Vendors. I am somewhat confident that I can find someone to set up something, but it would be good to discuss how to maintain this best in the longer term.
+- Discussion on CI: It would be nice to test the offloading support on at least hardware from all 3 major GPU Vendors.
```

Comment:
This all sounds non-normative. I think we should simplify to

> Compiler team: Minimal reviews to ensure the exposed LLVM features do not negative affect non-users. 

We could include a note on "optional CI support" but I don't think we have any way to test on GPU?